simulariumio/tests/converters/test_cytosim_converter.py

```diff
@@ -322,38 +336,49 @@
                     "fibers": CytosimObjectInfo(
                         filepath="simulariumio/tests/data/cytosim/"
                         "aster_pull3D_couples_actin_solid_3_frames/fiber_points.txt",
-                        agents={
-                            1: CytosimAgentInfo(name="microtubule", radius=0.01),
-                            2: CytosimAgentInfo(name="actin", radius=0.01),
+                        display_data={
+                            2: DisplayData(
+                                name="actin",
+                                radius=0.01,
+                                color="#ffc100",
+                            ),
                         },
                     ),
                     "solids": CytosimObjectInfo(
                         filepath="simulariumio/tests/data/cytosim/"
                         "aster_pull3D_couples_actin_solid_3_frames/solids.txt",
-                        agents={
-                            1: CytosimAgentInfo(name="aster", radius=0.1),
-                            2: CytosimAgentInfo(name="vesicle", radius=0.1),
+                        display_data={
+                            1: DisplayData(name="aster"),
+                            2: DisplayData(name="vesicle", radius=0.1),
                         },
                     ),
                     "singles": CytosimObjectInfo(
                         filepath="simulariumio/tests/data/cytosim/"
                         "aster_pull3D_couples_actin_solid_3_frames/singles.txt",
-                        agents={
-                            1: CytosimAgentInfo(name="dynein", radius=0.01),
-                            2: CytosimAgentInfo(name="kinesin", radius=0.01),
+                        display_data={
+                            1: DisplayData(name="dynein", radius=0.01),
+                            2: DisplayData(
+                                name="kinesin",
+                                radius=0.01,
+                                display_type="PDB",
+                                url="https://files.rcsb.org/download/3KIN.pdb",
+                                color="#0080ff",
+                            ),
                         },
                     ),
                     "couples": CytosimObjectInfo(
                         filepath="simulariumio/tests/data/cytosim/"
                         "aster_pull3D_couples_actin_solid_3_frames/couples.txt",
-                        agents={1: CytosimAgentInfo(name="motor complex", radius=0.02)},
+                        display_data={
+                            1: DisplayData(name="motor complex", radius=0.02)
+                        },
                         position_indices=[3, 4, 5],
                     ),
                 },
             ),
             {
                 "trajectoryInfo": {
-                    "version": 2,
+                    "version": 3,
```

Comment:
it would be good to make a CURRENT_VERSION constant held in the general tests folder that you import to all these files so you don't have to edit each one each time we bump the version

Reply:
you could even put it in here: https://github.com/allen-cell-animated/simulariumio/blob/main/simulariumio/__init__.py

and have a get_file_format_version. Or you could just put it in constants.py. either way, you shouldn't be writing it in every one of these files

Reply:
yeah added it to constants, there's a separate version for each block